src/components/MAPI/organizations/OrganizationDetailGeneral/utils.ts

```diff
@@ -376,24 +397,31 @@ export async function fetchAppsSummary(
   // The key is the app name, and the value is the number of deployments.
   const apps: Record<string, number> = {};
 
-  try {
-    // Get all apps that belong to all clusters.
-    const appLists = await Promise.all(
-      clusters.map((cluster) => {
-        return applicationv1alpha1.getAppList(httpClientFactory(), auth, {
-          namespace: cluster.metadata.name,
-        });
-      })
-    );
+  // Get all apps that belong to all clusters.
+  const responses = await Promise.allSettled(
+    clusters.map((cluster) => {
+      return applicationv1alpha1.getAppList(httpClientFactory(), auth, {
+        namespace: cluster.metadata.name,
+      });
+    })
+  );
```

Comment:
While we're still working with app resources in WC namespaces, we can't assume that a user automatically has access to every WC namespace for clusters within an organization. This part is changed so that when we fetch `apps`, we don't throw an error in case we couldn't fetch `apps` from one WC namespace due to permissions-related errors (status 403).